features/game/components/GamePanel.tsx

```diff
@@ -1,17 +1,39 @@
 import { Button } from '@/components/Elements/Button'
+import Select, { SelectOption } from '@/components/Elements/Select'
 import Panel from '@/components/Panel'
 import { Game } from '@prisma/client'
 import Link from 'next/link'
-import { useEffect, useState } from 'react'
+import { useEffect, useRef, useState } from 'react'
 import toast from 'react-hot-toast'
-import QRCode from 'react-qr-code'
 import { useSWRConfig } from 'swr'
 import { deleteGame } from '../api/deleteGame'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { deleteToast } from '@/components/Toasts'
+import { DocumentDownloadIcon } from '@heroicons/react/solid'
+import 'node-self'
```

Comment:
Where do we need `node-self`?

Reply:
see commit 986eb26d

Reply:
Adding to 
> see commit [986eb26](https://github.com/re-quest-team/request/commit/986eb26de8ebaa7d44145b59e68cea8443c5186b)

Module 'qr-code-styling' uses 'self' which is not available on Server-side, causing 'Server Error: ReferenceError: self is not defined'.  
After trying supposed fixes that did not work, I was able to fix this by using ['node-self'](https://www.npmjs.com/package/node-self), importing it is enough for it to work.  
It's very light weight and has no dependencies.

Reply:
Alright :+1: